test/orm/test_query.py

```diff
@@ -633,6 +633,42 @@ class Foo(object):
 
 
 class GetTest(QueryTest):
+    def test_get_composite_pk_keyword_based_no_result(self):
+        CompositePk = self.classes.CompositePk
+
+        s = Session()
+        assert s.query(CompositePk).get({"i": 100, "j": 100}) is None
```

Comment:
is_(s.query(CompositePK).get(....), None)

`is_()` is available from sqlalchemy.testing if it's not imported already